addon/templates/components/basic-dropdown-content.hbs

```diff
@@ -28,6 +29,31 @@
           {{yield}}
         </Element>
       {{/let}}
+      {{else}}
```

Comment:
It's a shame that we have to repeat the block. @ef4 is there a less verbose way of making the embroider gods happy?